src/account.cairo

```diff
@@ -3,9 +3,12 @@ mod dual_account;
 mod interface;
 
 use account::Account;
-use account::QUERY_VERSION;
-use account::TRANSACTION_VERSION;
 use interface::AccountABIDispatcher;
 use interface::AccountABIDispatcherTrait;
 use interface::AccountCamelABIDispatcher;
 use interface::AccountCamelABIDispatcherTrait;
+
+const TRANSACTION_VERSION: felt252 = 1;
+
+// 2**128 + TRANSACTION_VERSION
+const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457;
```

Comment:
although i like "cleaning" the `use` area in `account/account.cairo`, i kind of favor having these associated to the actual account impl module as source of truth. for example, in the future i expect us to have different accounts which might have different tx or query versions in the future.

Reply:
Maybe is worth moving them into the module instead of the file? Just a thought, no strong opinion.

Reply:
same, no strong opinion between module and file, as long as it's not the module's root

Reply:
Let me know what you think about the update.